react-multiple-pages/my-app/src/components/Header.tsx

```diff
@@ -0,0 +1,24 @@
+import { Outlet } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+
+export function Header() {
+  return (
+    <div>
+      <nav className="px-4 text-white bg-gray-900">
+        <ul>
+          <li className="inline-block py-2 px-4">
+            <Link to="/about" className="text-white">
+              About
+            </Link>
+          </li>
+          <li className="inline-block py-2 px-4">
+            <Link to="/dashboard" className="text-white">
+              Dashboard
+            </Link>
+          </li>
+        </ul>
+      </nav>
+      {<Outlet></Outlet>}
```

Comment:
When adding components to JSX, there is no need to use the `{}` around them. Also, since there are no children, you can use the self-closing version, so it would just look like `<Outlet />`.